prisma/seed.ts

```diff
@@ -147,43 +150,43 @@ const createParks = async () => {
       image: 'parks/tschutterplatz-beim-stadion.png',
       qr: 'not-yet-defined-6',
       coordinates: {
-        latitude: 47.503531,
-        longitude: 9.734675,
+        latitude: 47.50383807029892,
+        longitude: 9.73433812513105,
       },
     },
 
     {
       id: 8,
       name: 'Schulsportplatz VS Augasse',
-      address: 'Rotfarbgasse 14a, 6900 Bregenz',
+      address: 'Augasse 6, 6900 Bregenz',
       image: 'parks/schulsportplatz-vs-augasse.png',
       qr: 'not-yet-defined-7',
       coordinates: {
-        latitude: 47.500276,
-        longitude: 9.738716,
+        latitude: 47.49995266468694,
+        longitude: 9.736555724858727,
       },
     },
 
     {
       id: 9,
       name: 'Schulplatz VS Rieden',
-      address: 'Rotfarbgasse 14a, 6900 Bregenz',
+      address: 'Michael-Gaismayr-Straße 32, 6900 Bregenz',
       image: 'parks/schulsportplatz-vs-augasse.png',
       qr: 'not-yet-defined-8',
       coordinates: {
-        latitude: 47.49278,
-        longitude: 9.723823,
+        latitude: 47.49298641752309,
+        longitude: 9.722950393600897,
```

Comment:
des alles hät i eher in production seed gmacht

Reply:
döt hosches du jo eh scho gmacht i hob nur da rest im normala seed ergänzt